assets/model_monitoring/components/src/genai_token_statistics_compute_metrics/constants.py

```diff
@@ -16,7 +16,9 @@
 MODEL_COMPLETION_COUNT = "model_completion_count"
 
 # Constants for attribute keys
-COMPLETION_COUNT_KEY = "llm.token_count.completion"
-PROMPT_COUNT_KEY = "llm.token_count.prompt"
-TOTAL_COUNT_KEY = "     llm.token_count.total"
-MODEL_KEY = "llm.model"
+COMPLETION_COUNT_KEY = ["llm.token_count.completion", "llm.usage.completion_tokens"]
```

Comment:
Just for my understanding, are we expecting only one of these keys to appear for each span? If not, what would happen if multiple of these exist in span attributes does it matter which one we count?

Reply:
yes only one of these is expected to handle multiple runtime. also even if multiple of them exists which is not possible, we take value from only one making sure we dont compute the value wrong.